proj/src/main/webapp/static/chat.html

```diff
@@ -27,7 +27,17 @@
     <body onload="init()">
         <div class="container screen chat-page full-height">
             <div class="row">
+
             <div class="col">
+
+                <aside id="user-profile" class="display-none">
```

Comment:
Maybe I'm a little late saying this, but it feels weird to me to have a class just to set display: none.  Classes usually correspond to the kind of thing that needs to be styled (title, sidebar, chat message) rather than particular styling details (color, padding, display).  Is there a reason there's a class for this rather than just setting `style="display: none"`?

Reply:
fixed!